docker_update_latest.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# This script is intended to push a given tag to docker hub
+# and tag it as latest.  The idea is that this is part of 
+# the container CI workflow.  After testing a particular
+# tag, you can then declare it as the version to be used 
+# in CI
+
+if [[ $# -lt 1 ]]; then
+   echo "usage: docker_update_latest.sh <name> <tag>"
+   exit 1
+fi
+
+CNAME=${1:-"gnu-openmpi-dev"}
+TAG=${2:-"ecsync"}
+
+if [[ $(echo ${CNAME} | cut -d- -f1) = "intel19" ]]; then
+#if [[ $(echo ${CNAME} | cut -d- -f1) =~ "intel*" ]]; then
```

Comment:
Could be removed, it would be available via git log

Reply:
Thanks @rickgrubin for catching that - that was just a stopgap for the sprint that I meant to change.